### PR TITLE
Front-end UI, admin UI cleanup, python cron-jobs folder collector and bug fixes

### PR DESCRIPTION
**Pull request by fmalina**
Simplify Admin UI by collapsing less used features
Changed monitor_description to description so all jobs can have a description explaining them, not just monitors
Fixed monitor admin page where all jobs were show monitors due to get_queryset, now only is_monitor will be shown
Fixed a bug with duration graph page broken due to static templatetags
Removed arbitrary ID from job name
Fix some typos in code and PEP8

### CODE REVIEW

**Comment by fmalina** [comment]
Hi @chrisspen I'd like to contribute an actual frontend grade UI for job management on top of this "fixes only" PR when merged along with a job collector/sync command for a collection of python jobs. It's extracted from an enterprise ETL project I did and on my fork, I'll make another PR when this one is merged if this would be of interest. Thank you

**Comment by fmalina** [comment]
Oh, it's all there actually as it's on the fork now, fair enough 👍

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def get_reqs(*fns):
 
 
 setup(
-    name="django-chroniker",
```
Comment (chrisspen) [inline setup.py:29]: Is there any specific reason for this? This is a Django-specific project, not a general purpose Python package, so the convention when naming is to prefix such packages like "django-<name>".

**Comment by chrisspen** [comment]
Sorry, I didn't notice this when you first opened the PR.

File: chroniker/models.py [hunk 2]
```diff
@@ -298,28 +298,28 @@ def due_with_met_dependencies(self, jobs=None):
             valid = True
 
             if job.check_is_running():
-                #print('Skipping job %i (%s) which is already running.' % (job.id, job))
+                # print('Skipping job %i (%s) which is already running.' % (job.id, job))
```
Comment (chrisspen) [inline chroniker/models.py:301]: Note, you don't need to waste time reformatting dead code like this. It just adds clutter to the PR. These should just be removed.

**Comment by fmalina** [comment]
It's a real shame, this is good functionality and made it chroniker usable for my use case at a time

**Pull request closed without merging**